bank/services/stoneBaas/baseStoneBaas.go

```diff
@@ -56,13 +56,23 @@ func getValue(percentageStr string) int64 {
 	return value
 }
 
+func getDiscountSeparator(index, length int) string {
+	isTheLastDiscount := length == index+1
```

Comment:
aqui mudaria com esse [comentário](https://github.com/mundipagg/boleto-api/pull/316/files#r1300266119) 😀 
```suggestion
	isTheLastDiscount := length == index
```